pkg/providers/providerVars.yaml

```diff
@@ -0,0 +1,4 @@
+- name: openai
```

Comment:
These could be just constants in the corresponding openai & cohere packages. We could use them as defaults to fill our configs but also, we could allow them to be overridden via configs if that makes sense.